upcloud/service/service.go

```diff
@@ -707,7 +711,7 @@ func (s *Service) GetTags() (*upcloud.Tags, error) {
 }
 
 // Wrapper that performs a GET request to the specified location and returns the response or a service error
-func (s *Service) basicJSONGetRequest(location string) ([]byte, error) {
+func (s *Service) basicFutureGetRequest(location string) ([]byte, error) {
```

Comment:
Why future and not just basicGetRequest?

Reply:
And why change this at all?

Reply:
Hi,

This is a bit of a hangover from the way I did the port. `basicFutureGetRequest` was for the `1.3` API and `basicJSONGetRequest` was for the `1.2` API.

I removed all references of `future` in the code once I finished but I missed this one. I spotted it yesterday and it's already fixed in the next batch of changes.

Would you like me to fix it now or are you happy to wait for the next PR?

Thanks,

Stuart

Reply:
It's ok, for the next PR. 👍